### PR TITLE
Finally all MDN Web Docs

### PR DESCRIPTION
**Pull request by VyrekXD**
All the web docs of MDN please accept this :C
LTK7KoLU
f6q7jcRV
TpdyCafH

### CODE REVIEW

File: websites/M/MDN Web Docs/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "Vyrek",
+      "id": "538421122920742942"
+    },
+    "url": "developer.mozilla.org",
+    "description": {
+      "en": "Shows the things that you are seeing at developer.mozilla.org",
```
Comment (Bas950) [inline websites/M/MDN Web Docs/dist/metadata.json:9]: Need the description of the website

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/MDN Web Docs/dist/metadata.json (hunk 1 above)
Comment (VyrekXD) [reply websites/M/MDN Web Docs/dist/metadata.json:9]: Let me put it

**Review by Bas950 — DISMISSED** [review]

File: websites/M/MDN Web Docs/presence.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
+const presence = new Presence({
+    clientId: "786770326234464256"
+}), stamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+    const data: PresenceData = {
+        startTimestamp: stamp
+    }, paths = document.location.pathname.split('/');
+    paths.splice(0, 1);
```
Comment (ririxidev) [inline websites/M/MDN Web Docs/presence.ts:9]: this is unnecessary lol, you still use paths[1] to get paths, so why you remove this blank space? You only make this presence longer for no reason

File: websites/M/MDN Web Docs/presence.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            paths.splice(0, 4);
+            data.details = `${tech}: Looking at ${paths[0]}`;
+            if(paths[1]){
+                paths.splice(0, 1);
+                data.state = `Topic: ${paths.join(', ')}`;
+            }
+        } else {
+            data.details = `Looking at ${tech}`;
+        }
+    }
```
Comment (ririxidev) [inline websites/M/MDN Web Docs/presence.ts:99]: you could use switch for it :)

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/M/MDN Web Docs/presence.ts (hunk 3 above)
Comment (VyrekXD) [reply websites/M/MDN Web Docs/presence.ts:99]: thank u

File: websites/M/MDN Web Docs/presence.ts (hunk 2 above)
Comment (VyrekXD) [reply websites/M/MDN Web Docs/presence.ts:9]: okaaay

**Review by ririxidev — CHANGES_REQUESTED** [review]
upload new proofs of working presence and ready to accept :)

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload new proof of it working.

**Comment by VyrekXD** [comment]
there is the proof xd

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**